index.ts

```diff
@@ -34,30 +32,15 @@ const open = require('open');
 const path = require('path');
 import * as pluralize from 'pluralize';
 const commander = require('commander');
-const read = require('read-file');
 const readMultipleFiles = require('read-multiple-files');
 import * as recursive from 'recursive-readdir';
 import { Spinner } from 'cli-spinner';
-const splitLines = require('split-lines');
 import * as url from 'url';
 const readline = require('readline');
 const logging = require('@google-cloud/logging');
 const chalk = require('chalk');
 const { prompt } = require('inquirer');
-import { DOT, PROJECT_NAME, PROJECT_MANIFEST_BASENAME } from './src/utils.js';
-
-// Clasp settings file (Saved in ~/.clasprc.json)
-interface ClaspSettings {
-  access_token: string;
-  refresh_token: string;
-  token_type: string;
-}
-// Project settings file (Saved in .clasp.json)
-interface ProjectSettings {
-  scriptId: string;
-  rootDir: string;
-  projectId: string;
-}
+import { DOT, PROJECT_NAME, PROJECT_MANIFEST_BASENAME, ClaspSettings, ProjectSettings, DOTFILE } from './src/utils.js';
```

Comment:
We should limit line length in the future. #138

Reply:
Agreed